blog-highlights/demystifying-appflowy-editors-codebase.md

```diff
@@ -803,7 +800,7 @@ expect(selection.end, Position(path: [0], offset: pattern.length));
 ```
 
 
-That's it for this section. I urge the reader to look at more tests directly from the source code.
+For additional testing examples, please review the `appflowy_editor\test` directory. *TODO: provide suggestions that you think are particularly good models to follow as far as coverage and completeness go*
```

Comment:
sure, add the following: 

Review the following tests which are good models to follow as far as coverage and completeness go:
 - test\new\block_component\table_block_component
 - test\new\service\shortcuts\command_shortcut_events\backspace_command_test.dart
 - test\core